### PR TITLE
`py310` testing on the CI

### PR DESCRIPTION
**Pull request by dmitryduev**
Description
-----------
CircleCI now offers a `python` 3.10 image, so I enabled the `unit-s_base-lin-py310` `tox` testenv.

Testing
-------
On the CI. It miraculously failed the first time I ran it. 

Got a bizarre failure locally:
image

Fails consistently, will look into it...

Checklist
-------
- Include reference to internal ticket "Fixes WB-NNNN" (and github issue "Fixes #NNNN" if applicable)

### CODE REVIEW

**Review by vanpelt — APPROVED** [review]
🥳

File: .circleci/config.yml [hunk 1]
```diff
@@ -892,11 +892,10 @@ workflows:
           name: "unit-s_base-lin-py39"
           image: "python:3.9"
           toxenv: "py39"
-# Enable when circle offers python 3.10 image
-#      - test:
-#         name: "unit-s_base-lin-py310"
-#         image: "python:3.10"
-#         toxenv: "py310"
+      - test:
```
Comment (kptkin) [inline .circleci/config.yml:895]: should we add matrix logic for these unit-s_base jobs as well?

**Review by kptkin — APPROVED** [review]
Glad it worked (the second time 🤔 ) haha

File: .circleci/config.yml (hunk 1 above)
Comment (dmitryduev) [reply .circleci/config.yml:895]: probably not as the "matrix" will be too sparse :) let's revisit before the next release - I'm cooking a PR that will reorganize testing a bit to speed things up by converting/splitting our parallel jobs into a proper dag with certain jobs depending on each other.

**Comment by dmitryduev** [comment]
Ok, I confirm that the issue pops up after updating `scikit-learn` to >1, so probably the same as what wandb/sweeps has seen -- will see how easy it is to fix.

**Comment by dmitryduev** [comment]
@kptkin @vanpelt turns out this is a purely `sklearn`-related issue, so I am going to merge this and open a dedicated ticket/PR.

**Pull request merged**